### ISSUE TITLE
Cannot connect without INFO permission

### ISSUE THREAD

**Issue by moreinhardt** [redis/RedisDesktopManager#5270]
**Describe the bug**
I don't understand why this GUI needs permission to run `INFO` on the redis server just to connect.

**To Reproduce**
1. Configure ACLs to remove the permission to run `INFO` from the user you use to connect. In this example I disallow all `dangerous` and `admin` commands. The former includes the `INFO` command.
```
ACL SETUSER default on >mypassword +@all -@dangerous -@admin
```
2. Try to connect to the server using RESP.app.
3. While the actual connection _is_ established the GUI just freezes and after some time I get a pop-up with the message `Cannot connect to server 'localhost 6379'. Check log for details.`. I never see any keys. The RESP.app log shows:
```
Connection: localhost 6379 > connected
Connection: localhost 6379 > [runCommand] AUTH *******
Connection: localhost 6379 > [runCommand] PING
Connection: localhost 6379 > [runCommand] INFO ALL
Connection: Cannot refresh server info: NOPERM this user has no permissions to run the 'info' command
```
**Expected behavior**
I can connect to the server and interact normally with the keys. Just not see any server infos. Maybe I'm just missing a setting where I can disable this `INFO ALL` call?

**Environment (please complete the following information):**
 - OS & version: Ubuntu 20.04
 - Redis-Server version: 7.0.5

**Comment by uglide** [comment]
@moreinhardt Thanks for the bug report. Your point is valid and RESP.app should be able to operate with the blocked `INFO` command. Fix was added and will be released as part of the 2022.5 version.

PS: Your ACL command is not correct. You need to add `allkeys` to be able to read keys in RESP.app

**Comment by moreinhardt** [comment]
Wow! This is an insanely fast response, thank you! :smiley: 

>PS: Your ACL command is not correct. You need to add `allkeys` to be able to read keys in RESP.app

Actually, according to [these docs](https://redis.io/docs/manual/security/acl/#configure-acls-with-the-acl-command) the default user can access all keys per default. So I didn't need to add `allkeys` or `~*`. :thinking:
